build.assets/windows/build.ps1

```diff
@@ -456,9 +494,6 @@ function Build-Artifacts {
     )
     Write-Host "Starting build process for Teleport $TeleportVersion..."
 
-    # Setup for builds
-    Install-BuildRequirements -InstallDirectory "C:\toolchains" -TeleportSourceDirectory "$TeleportSourceDirectory"
-
```

Comment:
Was this intentional?

Reply:
Yes, we used to install Go as part of the "build binaries" step. But now we need Go for the "generate version info" step so this was moved out and called ahead of time (see linked e PR).